### PR TITLE
TINY-8939: Add allow attribute as a valid attribute for the iframe element in the Schema

### PR DESCRIPTION
**Pull request by hamza0867**
Related Ticket: TINY-8939

Description of Changes:
* Add `allow` attribute as a valid attribute for the `iframe` element in the Schema

Pre-checks:
* [x] Changelog entry added
* [x] ~Tests have been added (if applicable)~
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] ~Docs ticket created (if applicable)~

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/core/main/ts/api/html/Schema.ts [hunk 1]
```diff
@@ -65,8 +65,8 @@ export interface SchemaElement extends ElementRule {
   pattern?: RegExp;
 }
 
-export interface SchemaMap { [name: string]: {} }
-export interface SchemaRegExpMap { [name: string]: RegExp }
+export interface SchemaMap {[name: string]: {}}
```
Comment (lnewson) [inline modules/tinymce/src/core/main/ts/api/html/Schema.ts:68]: These whitespace changes should have been done. In fact very soon eslint will fail with this change.

File: modules/tinymce/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `text_patterns_lookup` option to provide additional text patterns dynamically #TINY-8778
 - New promotion element has been added to the UI and can be disabled using the new `promotion` option #TINY-8840
 - New `format_noneditable_selector` option to specify the `contenteditable="false"` elements that can be wrapped in a format. #TINY-8905
+- `allow` attribute is now a valid attribute od the `iframe` schema
```
Comment (lnewson) [inline modules/tinymce/CHANGELOG.md:13]:
```suggestion
- `allow` is now a valid attribute for the `iframe` schema. #TINY-8939
```

**Comment by lnewson** [comment]
@hamza0867 the PR title also needs fixing as remember it's used for the final commit message.

- [ ] Fix PR title

File: modules/tinymce/CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `text_patterns_lookup` option to provide additional text patterns dynamically #TINY-8778
 - New promotion element has been added to the UI and can be disabled using the new `promotion` option #TINY-8840
 - New `format_noneditable_selector` option to specify the `contenteditable="false"` elements that can be wrapped in a format. #TINY-8905
+- `allow` is now a valid attribute for the `iframe` schema. #TINY-8939
```
Comment (spocke) [inline modules/tinymce/CHANGELOG.md:13]: Maybe. Having a back ticks item as the first thing looks a bit weird.

```suggestion
- Added `allow` as valid attribute for the `iframe` element in the editor schema. #TINY-8939
```

**Comment by lnewson** [comment]
> @hamza0867 the PR title also needs fixing as remember it's used for the final commit message.
> 
> * [ ]  Fix PR title

@hamza0867 this still hasn't been fixed yet...

**Review by TheSpyder — APPROVED** [review]

**Pull request merged**